docs/about.md

```diff
@@ -159,18 +159,38 @@ The following are our highest priority, non bug fix features:
 
 ## Team
 
-The DoneJS family of technologies is built by 100s of contributors:
+The DoneJS family of technologies is built by hundreds of contributors just like you! Our [contributing contribution guide] includes information about reporting bugs, finding ways to contribute, submitting new code, and more!
 
+If you’d like to take your commitment to DoneJS or its sub-projects to the next level, you can join our core team. To become a part of the core team, you simply have to:
+
+- [Email](mailto:contact@bitovi.com) the core team expressing your interest.
+- Attend the weekly _DoneJS Contributors_ meeting twice a month. [DoneJS Calendar](https://www.google.com/calendar/embed?src=jupiterjs.com_g27vck36nifbnqrgkctkoanqb4%40group.calendar.google.com&ctz=America/Chicago)
+- Make one small contribution, even a spelling correction, each month.
 
 ### Core team
 
-To become a contributor to DoneJS or its sub-projects, you simply have to:
+The core team is made up of both part-time and full-time contributors.
 
-- [Email](mailto:contact@bitovi.com) the core team expressing your interest.
-- Attend the weekly _DoneJS Contributors_ meeting twice a month. [DoneJS Calendar](https://www.google.com/calendar/embed?src=jupiterjs.com_g27vck36nifbnqrgkctkoanqb4%40group.calendar.google.com&ctz=America/Chicago)
-- Make one small contribution, even a spelling correction, a month.
+<div class="core-team-member">
+  <img class="member-avatar" src="https://avatars2.githubusercontent.com/u/2445805?v=3&s=400" alt="Kevin Dillon profile picture" />
+  <h4>Kevin Dillon</h4>
+  <p>
+    Kevin is a senior infrastructure engineer at The MathWorks in the Web and Mobile Tools team.
+    He focuses on the development of FuncUnit and Syn.
```

Comment:
@kdillon, let me know if you’d like something else as your bio for this page (or if for any reason you’d prefer not to be on it at all): https://donejs.com/About.html#section=section_Team